test/src/test_suites/smoke-test.js

```diff
@@ -14,7 +14,7 @@ createTestCafe('localhost', 1337, 1338)
             "src/test_cases/edx-schools-frontend/test-school-inbox.js","src/test_cases/edx-schools-frontend/test-school-message-display.js"])
 */            .src(["src/test_cases/edx-new-user-invite/","src/test_cases/edx-schools-frontend/test-new-message.js","src/test_cases/edx-schools-frontend/test-school-message-display.js",
             "src/test_cases/edx-user-activation/school/","src/test_cases/edx-user-activation/district/", "src/test_cases/edx-districts-frontend/test-district-contacts.js",
-                "src/test_cases/edx-districts-frontend/test-school-contacts.js","src/test_cases/edx-schools-frontend/test-school-details-edit.js"])
+                "src/test_cases/edx-districts-frontend/test-school-contacts.js","src/test_cases/edx-schools-frontend/test-school-details-edit.js","src/test_cases/edx-schools-frontend/test-school-details-edit-no-admin.js"])
           .run();
```

Comment:
I think we want this test in regression.js. I don't think smoke-test gets run daily.